targets/targets.json

```diff
@@ -668,367 +668,321 @@
         "extra_labels_remove": ["FRDM"],
         "supported_form_factors": []
     },
+    "FAMILY_STM32": {
+        "inherits": ["Target"],
+        "core": "",
```

Comment:
You should not add this line.  Assigning core like this can cause problems if a child incorrectly does not define it's core attribute. Further, each child _must_ define the core attribute to function, overriding this one.